pillar/fluentd/ocw_mirror.sls

```diff
@@ -0,0 +1,65 @@
+{% from "fluentd/record_tagging.jinja" import record_tagging with context %}
+{% set pos_filename = '/data2/ocwmirror_logs.pos' %}
+
+fluentd:
+  plugins:
+    - fluent-plugin-secure-forward
+  configs:
+    - name: ocwmirror
+      settings:
+        - directive: source
+          attrs:
+            - '@id': ocwmirror_update_log
+            - '@type': tail
+            - enable_watch_timer: 'false'
+            - tag: ocwmirror.update
```

Comment:
Maybe tag it as `ocw.mirror.update` so that it's in keeping with the other ocw log tags